### ISSUE TITLE
Support localised versions of the game

### ISSUE THREAD

**Issue by Stephen-Gates** [opendataboardgame/game#34]
Consider adjusting the structure of the repository to support localisations of the game.
Perhaps:
- folders for each localisation (by country / language?)
- a guide on what files need to be changed (or not touched)
- process to follow to test and contribute back

**Comment by urapoh** [comment]
Alsio consider using a crowd localization service, such as Transifex. If so, my team may provide russian localization.

**Comment by Stephen-Gates** [comment]
OK my take on files that need localising (ignoring translation) are: 
- events.csv
- tools.csv

Potential changes include: 
- £, pounds
- hedgehogs, snow, voting caravan, asparagus, mole, plane tree, High street
- A&E (Accident and Emergency)

Less important changes: 
- character names
- place names 
- organisation names

@ellenbroad your thoughts?

**Comment by ellenbroad** [comment]
@Stephen-Gates yes to all. Most of the character, place, organisation names in our version are names of real people we know (Sheridan is John Sheridan, Well's Woods and Peter's Pond are Peter Wells, Dodds Cooperative is Leigh Dodds etc) so happy for people to localise according to their own local data heroes and heroines :). If I add a top level folder to the repository titled Localisations, with a mark down file noting changes to be made and details for review, this would be enough?

**Comment by Stephen-Gates** [comment]
@ellenbroad that would be a good start. I wonder if code needs to change to support printing your local version of the cards?

Trying to get a group together in Australia to localise.

**Comment by ellenbroad** [comment]
@JeniT is the best person to talk to about that.

**Comment by ap-Codkelden** [comment]
Hi all!

Is there any new rules for licolization? 
I want localize the game into Ukrainian, can I still use information described above?

Thanks!

**Comment by Dramat88** [comment]
Hello, I found this old thread because I am looking into translating/relocating the game. 
Did you manage to create the folder @ellenbroad ? Was there any code that needed to be changed @JeniT ? 
And did you create your localization @Stephen-Gates ?  
I look forward to hearing from you - thank you very much!
